packages/asana/src/Adaptor.js

```diff
@@ -233,39 +168,122 @@ export function createTask(params, callback) {
+          // projects and workspace ids should not be included to update
+          const { projects, workspace, ...remainingData } = data;
+          return updateTask(matchingTask.gid, remainingData, callback)(state);
         } else {
           console.log('No matching task found. Performing create.');
           return createTask(data, callback)(state);
         }
-      });
+      }
+    );
+  };
+}
+
+/**
+ * Options provided to the createTaskStory request
+ * @typedef {Object} StoryOptions
+ * @property {object} story - The story to create, It represents an activity associated with an object in the Asana system.
+ * @property {string} [story.text] - The plain text of the comment to add. Cannot be used with html_text.
+ * @property {string} [story.html_text] - Opt In. HTML formatted text for a comment. This will not include the name of the creator.
+ * @property {boolean} [story.is_pinned] - Conditional. Whether the story should be pinned on the resource.
+ * @property {string} [story.sticker_naem] - The name of the sticker in this story. `null` if there is no sticker.
+ * @property {array} [opt_fields] - Opt In. This endpoint returns a compact resource, which excludes some properties by default. To include those optional properties, set this query parameter to a comma-separated list of the properties you wish to include.
+ * @property {boolean} [opt_pretty] - Opt In. Provides the response in a “pretty” format. In the case of JSON this means doing proper line breaking and indentation to make it readable. This will take extra time and increase the response size so it is advisable only to use this during debugging.
+ */
+
+/**
+ * Create a story to a specific task.
+ * @public
+ * @example
+ * createTaskStory("1206933955023739", {
+ *   story: {
+ *     text: "This is a comment",
+ *     html_text: "<body>This is a comment</body>",
+ *     is_pinned: false,
+ *   },
+ * });
+ * @function
+ * @param {string} taskGid - Globally unique identifier for the task
+ * @param {StoryOptions} params - Story parameters
+ * @param {function} callback - (Optional) callback function
+ * @returns {Operation}
+ */
+export function createTaskStory(taskGid, params, callback) {
+  return state => {
+    const [resolvedTaskGid, resolvedParams] = expandReferences(
+      state,
+      taskGid,
+      params
+    );
+
+    const { story, opt_pretty = false, opt_fields = [] } = resolvedParams;
+    return commonRequest(
+      state,
+      `tasks/${resolvedTaskGid}/stories`,
+      {
+        body: { data: story },
+        query: { opt_fields, opt_pretty },
+        method: 'POST',
+      },
+      callback
+    );
+  };
+}
+
+/**
+ * Options provided to the Asana API request
+ * @typedef {Object} RequestOptions
+ * @property {object} body - Body data to append to the request.
+ * @property {object} query - An object of query parameters to be encoded into the URL.
+ * @property {string} method - The HTTP method to use
```

Comment:
We should say this defaults to get

Reply:
Update the method to `@property {string} method - The HTTP method to use. Defaults to 'GET'`